doc/installation.rst

```diff
@@ -11,10 +11,10 @@ Thanks!
 
 .. _issue tracker: https://github.com/spyder-ide/spyder/issues
 
-.. important::
+.. note::
 
-   Spyder now offers :ref:`standalone installers<standalone_installers_ref>` for Windows and macOS, making it easier to get up and running with the application without having to download Anaconda or manually install it in your existing environment.
-   While we still support Anaconda, we recommend this install method on those platforms to avoid most problems with package conflicts and other issues.
+   For most users on Windows and macOS, we recommend using our :ref:`standalone_installers_ref`, and for those on Linux or who need third-party Spyder plugins, which the standalone installers currently do not support, we advise using :ref:`installation-conda`.
+   Linux, plugin and package/environment management support in the standalone installers are currently under active development for future Spyder versions.
```

Comment:
I wanted the order to match the previous sentence, where the plugin support (which is the bigger deal there) is mentioned second. Its a bigger feature here, but its more of a nice to have than resolving a hard limitation in the installers like plugin support.

Also, I'm not sure its 100% set that it will come first, we could just add support for selecting custom plugins at install time, add a CLI or simplified minimal GUI for installing plugins before adding full package and env management for arbitrary packages and environments, which is a much larger problem space.

Reply:
> I wanted the order to match the previous sentence

Ok, I understand now the idea of this sentence. So, I'm ok with leaving it as it is.

> Also, I'm not sure its 100% set that it will come first, we could just add support for selecting custom plugins at install time, add a CLI or simplified minimal GUI for installing plugins before adding full package and env management for arbitrary packages and environments, which is a much larger problem space.

Package management will come first thanks to what Daniel and Juan Sebastian are doing.

Reply:
Okay, I see. In any case, if we change it here, we would need to change it in the previous sentence as well, and I'm really not sure it makes sense to put better support for custom data types in the variable explorer ahead of plugin support, especially when its not really that directly connected with the package and environment management anyway (as it refers to Spyder's own internal environment).

Reply:
As I said above, I'm fine with leaving this sentence as it is.